pkg/pillar/cmd/volumemgr/handlecontent.go

```diff
@@ -209,6 +262,14 @@ func updateContentTree(ctx *volumemgrContext, status *types.ContentTreeStatus) {
 
 func deleteContentTree(ctx *volumemgrContext, status *types.ContentTreeStatus) {
 	log.Infof("deleteContentTree for %v", status.ContentID)
+	RemoveBlobsFromContentTreeStatus(ctx, status, status.Blobs...)
```

Comment:
Ah not even checking the error.
Thus RemoveBlobsFromContentTreeStatus should definitely continue to remove all even if the lookup for one or more blobs fail. And no for it to return an error if the caller doesn't care.

Reply:
Handled.